mp/importer/batcher.py

```diff
@@ -26,9 +26,42 @@ def run_in_batches(iterable, end_batch_callback=None, batch_size=2, batch_start=
             if position_in_batch == 0: 							# ends completed batches
                 end_batch_callback()
                 ended = True
-        if max_batches is not None and current_batch == batch_start + max_batches:					# if gets maximum allowed batches, break 
+        if max_batches is not None and current_batch == batch_start + max_batches:	# if gets maximum allowed batches, break 
             break
     if ended is False:
         end_batch_callback()	
 
 
+def parse_arguments(args=None):		 						# receives a list of all the arguments and create a parser with only batcher args	
+    """ Parse arguments from user									 
+
+    This function parses user input and returns the object args that contains arguments
+    """
+    import argparse
+    parser = argparse.ArgumentParser(description='Pass arguments to batcher')				
+    parser.add_argument('--argument_1', dest='batcher_argument_1', type=int, help="1st argument")
+    parser.add_argument('--argument_2', dest='batcher_argument_2', type=int, help="2nd argument")
+    options = None
+    if args is not None: 
+        options = parser.parse_args(args)						# need to skip the first argument (script name) if this function receives full args (args[:1])
+    return options			 
+
+def get_batcher_args(options):									
+    """ Takes options and passes arguments to batcher
+
+    Gets arguments parsed by parse_arguments to be passed as  a dictionary of keywords (**kw) to run_in_batches
+    e.g. run_in_batches(iterable, end_batch, **get_batcher_args(options))
+    """ 
+    kw = dict()
```

Comment:
according to the argparse docs, you can use vars() function to create a dict of options from the options object